src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,26 @@
-export const UserInfo = () => {};
+import React from 'react';
+import { User } from '../../types/User';
+
+type Prop = {
+  user: User | null;
+};
+
+export const UserInfo: React.FC<Prop | null> = ({ user }) => (
```

Comment:
```suggestion
export const UserInfo: React.FC<Props> = ({ user }) => (
```
you should not do it